ovs/extensions/healthcheck/arakoon/arakooncluster_health_check.py

```diff
@@ -325,45 +330,46 @@ def verify_integrity(logger, arakoon_clusters=None):
             if ArakoonHealthCheck.MACHINE_DETAILS.machine_id not in cluster_info:
                 continue
 
-            tries = 1
-            max_tries = 2  # should be 5 but .nop is taking WAY to long
+            with volatile_mutex('ovs-healthcheck_arakoon-test_{0}'.format(cluster_name)):
```

Comment:
Arakoon collapse test tries to collapse local arakoons. By locking them cluster wide, you will have only tested the arakoons on this node and will report a false value back to the consumer.